internal/cmd/beta/server/start/start.go

```diff
@@ -0,0 +1,115 @@
+package start
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	iaasUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/spinner"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas/wait"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	serverIdArg = "SERVER_ID"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ServerId string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "start",
+		Short: "Start an existing server or allocates the server if deallocated",
+		Long:  "Start an existing server or allocates the server if deallocated.",
+		Args:  args.SingleArg(serverIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(
+				`Start an existing server with ID "xxx"`,
+				"$ stackit beta server start xxx",
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			serverLabel, err := iaasUtils.GetServerName(ctx, apiClient, model.ProjectId, model.ServerId)
+			if err != nil {
+				p.Debug(print.ErrorLevel, "get server name: %v", err)
+				serverLabel = model.ServerId
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			err = req.Execute()
+			if err != nil {
+				return fmt.Errorf("server start: %w", err)
+			}
+
+			// Wait for async operation, if async mode not enabled
+			if !model.Async {
+				s := spinner.New(p)
+				s.Start("Starting server")
+				_, err = wait.StartServerWaitHandler(ctx, apiClient, model.ProjectId, model.ServerId).WaitWithContext(ctx)
+				if err != nil {
+					return fmt.Errorf("wait for server starting: %w", err)
+				}
+				s.Stop()
+			}
+
+			p.Info("Server %q started!\n", serverLabel)
```

Comment:
Sincr the operation is not synchronous, we should only output `Started ...` if we have called the waiter. Otherwise it should say `Triggered start of ...` (or something similar). You can check e.g. https://github.com/stackitcloud/stackit-cli/blob/9324319c69c607028f0e2e3381348e0b861898f4/internal/cmd/postgresflex/instance/create/create.go#L295

Reply:
Also not sure about the `!` (just becasue I don't think it's present in other commands)

Reply:
Samr for other commands in the PR